test/integration/express/express.test.js

```diff
@@ -0,0 +1,269 @@
+                        'process.pid': expect.any(Number),
+                        'process.runtime.version': expect.stringMatching(/\d+\.\d+\.\d+/),
+                        'process.runtime.name': 'nodejs',
+                        'process.executable.name': 'node',
+                    },
+                },
+                attributes: {
+                    'http.method': 'GET',
+                    'http.target': '/invoke-requests',
+                    'http.flavor': '1.1',
+                    'http.host': expect.stringMatching(/localhost:\d+/),
+                    'http.scheme': 'http',
+                    'net.peer.ip': '::ffff:127.0.0.1',
+                    'http.request.query': '{}',
+                    'http.request.headers': expect.stringMatching(/\{.*\}/),
+                    'http.response.headers': expect.stringMatching(/\{.*\}/),
+                    'http.response.body': expect.jsonMatching(["animal", "career", "celebrity", "dev", "explicit", "fashion", "food", "history", "money", "movie", "music", "political", "religion", "science", "sport", "travel"]),
+                    'http.request.body': '{}',
+                    'http.route': '/invoke-requests',
+                    'express.route.full': '/invoke-requests',
+                    'express.route.configured': '/invoke-requests',
+                    'express.route.params': '{}',
+                    'http.status_code': 200,
+                },
+                status: {
+                    code: 1,
+                },
+                events: [],
+            });
+
+            expect(internalSpan).toMatchObject({
+                traceId: expect.any(String),
+                id: expect.any(String),
+                timestamp: expect.any(Number),
+                duration: expect.any(Number),
+                name: 'HTTP GET',
+                kind: 1,
+                attributes: {
+                    'http.url': expect.stringMatching(/http:\/\/localhost:\d+\/invoke-requests/),
+                    'http.host': expect.stringMatching(/localhost:\d+/),
+                    'net.host.name': 'localhost',
+                    'http.method': 'GET',
+                    'http.target': '/invoke-requests',
+                    'http.user_agent': 'axios/0.21.4',
+                    'http.flavor': '1.1',
+                    'net.transport': 'ip_tcp',
+                    'net.host.ip': '::ffff:127.0.0.1',
+                    'net.host.port': expect.any(Number),
+                    'net.peer.ip': '::ffff:127.0.0.1',
+                    'net.peer.port': expect.any(Number),
+                    'http.status_code': 200,
+                    'http.status_text': 'OK',
+                    'http.route': '/invoke-requests',
+                },
+                status: {
+                    code: 0,
+                },
+                events: [],
+            });
+            expect(clientSpan).toMatchObject({
+                traceId: expect.any(String),
```

Comment:
don't you prefer to have builders to this huge data, so the test will be more readable?
Maybe even just export it to a const?